SPECS/jasper/jasper.spec

```
@@ -1,46 +1,82 @@
+
+# NOTE: packages that can use jasper:
+# ImageMagick
+# netpbm
+
 Summary:        Implementation of the JPEG-2000 standard, Part 1
```

We can find these dependencies through a quick spec search or using package managers on running images. No need to mention this here, I think. Unless it's some special case.

```suggestion
Summary:        Implementation of the JPEG-2000 standard, Part 1
```

---

Done.